release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -265,6 +265,24 @@ For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#no
 
 With this release, you can increase the disk quota in etcd. This is a Technology Preview feature. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-increase-db_recommended-etcd-practices[Increasing the database size for etcd].
 
+[id="ocp-4-16-reserved-core-freq-tuning_{context}"]
+====  Reserved Core Frequency Tuning
+
+With this release, the Node tuning Operator supports setting CPU frequencies in the `PerformanceProfile` for reserved and isolated core CPUs.
+This is an optional feature that you can use to define specific frequencies. 
+The Node Tuning Operator then sets those frequencies by enabling the `intel_pstate` CPUFreq driver in the Intel hardware.
+You must follow Intel's recommendations on frequencies for FlexRAN-like applications, which require the default CPU frequency to be set to a lower value than the default running frequency.
```

Comment:
Thanks @sabbir-47 I've updated here. How does this look?

Reply:
looks good.